File: sofd-core-opus-integration/src/main/java/dk/sofd/opus/service/model/Phone.java
package dk.sofd.opus.service.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@Builder
@ToString
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(exclude = { "prime", "typePrime" })
public class Phone {
	private String master;
	private String masterId;
	private String phoneNumber;
	private String phoneType;
	private String visibility;
	private boolean prime;
	private boolean typePrime;
}

File: sofd-core-sd-integration/sofd-core-sd-integration/SOFD/Model/Phone.cs
using Newtonsoft.Json;

namespace DigitalIdentity.SOFD.Model
{
    public class Phone
    {
        [JsonProperty("master")]
        public string Master { get; set; }

        [JsonProperty("masterId")]
        public string MasterId { get; set; }

        [JsonProperty("phoneNumber")]
        public string PhoneNumber { get; set; }

        [JsonProperty("phoneType")]
        public string PhoneType { get; set; }

        [JsonProperty("visibility")]
        public string Visibility { get; set; }

        [JsonProperty("functionTypeId")]
        public int FunctionTypeId { get; set; }

        [JsonProperty("functionTypeName")]
        public string FunctionTypeName { get; set; }

        [JsonProperty("prime")]
        public bool Prime { get; set; }

        [JsonProperty("typePrime")]
        public bool TypePrime { get; set; }
    }
}
